docs/user-docs/apps.md

```diff
@@ -1,84 +1,147 @@
 # Jellyfin Apps
 
-A list of apps with support for Jellyfin.
+Apps connect your devices to your Jellyfin server and let you view your content on any supported device. You can find a list of apps, as well as their status, below.
 
-**Note**: Many of the official apps were forked from Emby and are not currently building and/or functional.
+**Note**: Many apps were direct forks of the latest available codebases of Emby apps, and some are not functional. 
```

Comment:
```suggestion
**Note**: Many apps were direct forks of the latest available codebases of Emby apps, and some are not functional yet. 
```

Reply:
Added "yet" and "or are deprecated" to mention those too. Fixing.